authentication/3.1/_index.md

```diff
@@ -25,15 +25,15 @@ The release aims to evolve the API in a smaller way to support the overall goals
 * [Jakarta Authentication 3.1 Specification Document](./jakarta-authentication-spec-3.1.pdf) (PDF)
 * [Jakarta Authentication 3.1 Specification Document](./jakarta-authentication-spec-3.1.html) (HTML)
 * [Jakarta Authentication 3.1 Javadoc](./apidocs)
-* [Jakarta Authentication 3.1 TCK](https://download.eclipse.org/jakartaee/authentication/3.1/jakarta-authentication-tck-3.1.0.zip)  ([sig](https://download.eclipse.org/jakartaee/authentication/3.1/jakarta-authentication-tck-3.1.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/authentication/3.1/jakarta-authentication-tck-3.1.0.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* [Jakarta Authentication 3.1 TCK](https://download.eclipse.org/jakartaee/authentication/3.1/jakarta-authentication-tck-3.1.1.zip)  ([sig](https://download.eclipse.org/jakartaee/authentication/3.1/jakarta-authentication-tck-3.1.1.zip.sig),  [sha](https://download.eclipse.org/jakartaee/authentication/3.1/jakarta-authentication-tck-3.1.1.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
   * [jakarta.authentication:jakarta.authentication-api:jar:3.1.0](https://central.sonatype.com/artifact/jakarta.authentication/jakarta.authentication-api/3.1.0/jar)
 
 # Compatible Implementations
 
  * [Epicyro 3.1.0](https://github.com/eclipse-ee4j/epicyro/releases/download/3.1.0-RELEASE/epicyro-3.1.0.jar)
- * [Eclipse Glassfish 8.0.0-M6](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-M6/glassfish-8.0.0-M6.zip) (JDK 21)
- * [Eclipse Glassfish 8.0.0-JDK-17-M6](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-JDK17-M6/) (JDK 17)
+ * [Eclipse Glassfish 8.0.0-M6](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-M6/glassfish-8.0.0-M7.zip) (JDK 21)
+ * [Eclipse Glassfish 8.0.0-JDK-17-M6](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-JDK17-M7/) (JDK 17)
```

Comment:
```suggestion
 * [Eclipse Glassfish 8.0.0-M6](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-M6/glassfish-8.0.0-M6.zip) (JDK 21)
 * [Eclipse Glassfish 8.0.0-JDK-17-M6](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-JDK17-M6/) (JDK 17)
```
I don't believe we should change the ratifying implementation URLs

Reply:
In the past I think we have been updating to the recent version that passes the TCK whenever we update the TCK.

I hear what you're saying, but we don't have a good answer for it now. M6 was used to ratify the release it self, M7 (and incidentally, M9) were used to test the 3.1.1 TCK on. Maybe we should list them both with information on which version passed what, but we haven't been doing that for other APIs before.

Reply:
I thought the point for updating any service release was to run with the same release that was done to ratify the implementation initially and make sure that it didn't cause that to fail because that would indicate that the service release update broke things.  I don't believe we should use a newer version of an implementation when validating a service release of a TCK.

If you end up wanting to upating it to M7 or M9 (again I think that is the wrong decision), it should be done completely.  In the update currently M6 is left in three places which would result in the URL not working for one of the links and the wrong description of the link for both of them.